src/Backend/Core/Js/ckeditor/ckeditor_php5.php

```diff
@@ -1,556 +1,556 @@
-		}
-
-		// Skip relative paths...
-		if (strpos($ckeditorPath, '..') !== 0) {
-			$out .= $this->script("window.CKEDITOR_BASEPATH='". $ckeditorPath ."';");
-		}
-
-		$out .= "<script type=\"text/javascript\" src=\"" . $ckeditorPath . 'ckeditor.js' . $args . "\"></script>\n";
-
-		$extraCode = "";
-		if ($this->timestamp != self::timestamp) {
-			$extraCode .= ($extraCode ? "\n" : "") . "CKEDITOR.timestamp = '". $this->timestamp ."';";
-		}
-		if ($extraCode) {
-			$out .= $this->script($extraCode);
-		}
-
-		$initComplete = $this->initialized = true;
-
-		return $out;
-	}
-
-	/**
-	 * Return path to ckeditor.js.
-	 */
-	private function ckeditorPath()
-	{
-		if (!empty($this->basePath)) {
-			return $this->basePath;
-		}
-
-		/**
-		 * The absolute pathname of the currently executing script.
-		 * Note: If a script is executed with the CLI, as a relative path, such as file.php or ../file.php,
-		 * $_SERVER['SCRIPT_FILENAME'] will contain the relative path specified by the user.
-		 */
-		if (isset($_SERVER['SCRIPT_FILENAME'])) {
-			$realPath = dirname($_SERVER['SCRIPT_FILENAME']);
-		}
-		else {
-			/**
-			 * realpath - Returns canonicalized absolute pathname
-			 */
-			$realPath = realpath( './' ) ;
-		}
-
-		/**
-		 * The filename of the currently executing script, relative to the document root.
-		 * For instance, $_SERVER['PHP_SELF'] in a script at the address http://example.com/test.php/foo.bar
-		 * would be /test.php/foo.bar.
-		 */
-		$selfPath = dirname($_SERVER['PHP_SELF']);
-		$file = str_replace("\\", "/", __FILE__);
-
-		if (!$selfPath || !$realPath || !$file) {
-			return "/ckeditor/";
-		}
-
-		$documentRoot = substr($realPath, 0, strlen($realPath) - strlen($selfPath));
-		$fileUrl = substr($file, strlen($documentRoot));
-		$ckeditorUrl = str_replace("ckeditor_php5.php", "", $fileUrl);
-
-		return $ckeditorUrl;
-	}
-
-	/**
-	 * This little function provides a basic JSON support.
-	 *
-	 * @param mixed $val
-	 * @return string
-	 */
-	private function jsEncode($val)
-	{
-		if (is_null($val)) {
-			return 'null';
-		}
-		if (is_bool($val)) {
-			return $val ? 'true' : 'false';
-		}
-		if (is_int($val)) {
-			return $val;
-		}
-		if (is_float($val)) {
-			return str_replace(',', '.', $val);
-		}
-		if (is_array($val) || is_object($val)) {
-			if (is_array($val) && (array_keys($val) === range(0,count($val)-1))) {
-				return '[' . implode(',', array_map(array($this, 'jsEncode'), $val)) . ']';
-			}
-			$temp = array();
-			foreach ($val as $k => $v){
-				$temp[] = $this->jsEncode("{$k}") . ':' . $this->jsEncode($v);
-			}
-			return '{' . implode(',', $temp) . '}';
-		}
-		// String otherwise
-		if (strpos($val, '@@') === 0)
-			return substr($val, 2);
-		if (strtoupper(substr($val, 0, 9)) == 'CKEDITOR.')
-			return $val;
-
-		return '"' . str_replace(array("\\", "/", "\n", "\t", "\r", "\x08", "\x0c", '"'), array('\\\\', '\\/', '\\n', '\\t', '\\r', '\\b', '\\f', '\"'), $val) . '"';
-	}
-}
+<?php
```

Comment:
Yeah, just replaced everything... is it bad?  I mean, it shouldn't pose any real problems, and when CK editor is updated, this will be gone...